LambdaEngine/Source/Rendering/RenderAPI.cpp

```diff
@@ -35,7 +35,7 @@ namespace LambdaEngine
 	{
 		GraphicsDeviceDesc deviceDesc = { };
 #ifdef LAMBDA_DEVELOPMENT
-		deviceDesc.Debug = true;
+		deviceDesc.Debug = false;
 #else
 		deviceDesc.Debug = false;
 #endif
```

Comment:
Should it be false on both case?

Reply:
Forgot to switch it back. It is necessary for both to be false when debugging using nsight graphics